src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const tBody = document.querySelector('tbody');
+
+Array.from(tBody.children).forEach((tr) => {
+  const secondTd = tr.children[1];
+
+  const lastTd = tr.lastElementChild;
+
+  const clonedTd = secondTd.cloneNode(true);
+
+  tr.insertBefore(clonedTd, lastTd);
+});
+
+function cloneColumn(wrapper, columnIndex) {
+  const tFoot = document.querySelector(wrapper);
+
+  const secondThFoot = tFoot.querySelectorAll('th')[columnIndex];
```

Comment:
Ensure that the `columnIndex` is within the bounds of the `th` elements in the `tFoot`. If `columnIndex` is out of bounds, this could lead to unexpected behavior or errors.